src/applications/check-in/components/pages/Appointments/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+
+import { useFormRouting } from '../../../hooks/useFormRouting';
+import { useGetCheckInData } from '../../../hooks/useGetCheckInData';
+import Wrapper from '../../layout/Wrapper';
+import { APP_NAMES } from '../../../utils/appConstants';
+import { makeSelectApp } from '../../../selectors';
+import { useUpdateError } from '../../../hooks/useUpdateError';
+
+const AppointmentsPage = props => {
+  const { router } = props;
+  const selectApp = useMemo(makeSelectApp, []);
+  const { app } = useSelector(selectApp);
+  const { t } = useTranslation();
+  const [isLoading, setIsLoading] = useState(true);
+  const { goToNextPage } = useFormRouting(router);
+  const { updateError } = useUpdateError();
+
+  const {
+    isComplete,
+    refreshCheckInData,
+    checkInDataError,
+  } = useGetCheckInData({
+    refreshNeeded: true,
+    router,
+    isPreCheckIn: app === APP_NAMES.PRE_CHECK_IN,
+  });
+
+  const handleClick = () => {
+    goToNextPage();
+  };
+
+  useEffect(
+    () => {
+      setIsLoading(!isComplete);
+      if (!isComplete) {
+        refreshCheckInData();
```

Comment:
using refreshCheckInData is not necessary in this page. You can get rid of that import and this if statement here